cognite/client/data_classes/data_modeling/data_types.py

```diff
@@ -73,7 +72,7 @@ def __load_unit_ref(data: dict) -> UnitReference | None:
     @classmethod
     def _load(cls, resource: dict[str, Any], cognite_client: CogniteClient | None = None) -> Self:
         type_ = resource["type"]
-        obj: Any
+        obj: Any  # This is a hack to avoid a mypy error
```

Comment:
I pushed a commit with an alternate solution (that you should feel free to reject 😉 )

Reply:
Not pretty that either, but prettier